connectors/src/connectors/notion/temporal/workflows.ts

```diff
@@ -204,14 +208,19 @@ export async function notionSyncWorkflow(
                 parentClosePolicy:
                   ParentClosePolicy.PARENT_CLOSE_POLICY_TERMINATE,
               })
-            )
+            ) as Promise<SyncWorkflowResult>
```

Comment:
The pqueue adds a " | void" to the Promise return, I don't know why. For now I made a cast, feedback welcome

Reply:
Def not good you'll have to work around it more safely

Reply:
Did you or @fontanierh already encounter this? if yes do you know the reason / implications? If no, I'll dig

Reply:
Found what it meant (void = exec timeout since we use the default throwOnTimeout false); will change the code accordingly & remove the cast